FirstProject/css/main.css

```diff
@@ -80,11 +84,66 @@ h2.description_section {
     margin-left: 300px;
 }
 
-/*.row_section {*/
-    /*margin-left:254px;*/
-    /*margin-right: 255px;*/
-/*}*/
-
 fieldset.field_section {
     text-align: center;
-}
\ No newline at end of file
+}
+
+legend {
+    text-align: left;
+    color: #03A9F4;
+}
+
+input[type='radio']{
+    width: 35px;
+    height: 23px;
+    border-radius: 15px;
+    top: -2px;
+    left: -1px;
+    position: relative;
+    background-color: #d1d3d1;
+    content: '';
+    display: inline-block;
+    visibility: visible;
+    border: 2px solid white;
```

Comment:
What background color are you using for the parent container? Is this border visible in that context?